### PR TITLE
Blog post: Wasm and CNCF updates

### LINKED ISSUE #19: Blog about recent mentions of Kubewarden
We have been recently featured twice on the new stack:

* https://thenewstack.io/what-is-webassembly/
* https://thenewstack.io/webassembly-aims-to-eliminate-the-file-system/

### PR DESCRIPTION
**Pull request by flavio**
Talk about the recent news in the CNCF/WebAssembly space.

This fixes #19

### CODE REVIEW

File: content/blog/2021/07/webassembly-is-coming-to-cloud-native.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+and
+[ORAS](https://oras.land/) (see [here](https://github.com/cncf/toc/issues/692)).
+
+Let's take a closer look at them!
+
+---
+
+{{< figure class="center" src="/images/krustlet-logo.svg" width="60%" alt="Krustlet logo">}}
+
+[Krustlet](https://krustlet.dev/) registers itself against a Kubernetes clusters
```
Comment (ereslibre) [inline content/blog/2021/07/webassembly-is-coming-to-cloud-native.md:66]:
```suggestion
[Krustlet](https://krustlet.dev/) registers itself against a Kubernetes cluster
```

File: content/blog/2021/07/webassembly-is-coming-to-cloud-native.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+as a Kubelet instance. However, the Krustlet Kubelet does not run regular containers,
+it instead executes WebAssembly code.
```
Comment (ereslibre) [inline content/blog/2021/07/webassembly-is-coming-to-cloud-native.md:68]:
```suggestion
it instead executes WebAssembly programs.
```

File: content/blog/2021/07/webassembly-is-coming-to-cloud-native.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+Krustlet leverages [OCI registries](https://opencontainers.org/) to distribute
+the WebAssembly modules to execute. This is exactly the same approach Kubewarden
+adopts to distribute its policies.
+
+As a matter of fact, Krustlet and Kubewarden teams are collaborating on the
+[oci-distribution](https://crates.io/crates/oci-distribution) crate.
+crate. This is the Rust library that Krustlet and Kubewarden use to retrieve
```
Comment (ereslibre) [inline content/blog/2021/07/webassembly-is-coming-to-cloud-native.md:76]:
```suggestion
[oci-distribution](https://crates.io/crates/oci-distribution) crate. This is the Rust
library that Krustlet and Kubewarden use to retrieve
```

File: content/blog/2021/07/webassembly-is-coming-to-cloud-native.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+WasmModules from OCI registries.
```
Comment (ereslibre) [inline content/blog/2021/07/webassembly-is-coming-to-cloud-native.md:77]:
```suggestion
Wasm modules from OCI registries.
```

or

```suggestion
WebAssembly modules from OCI registries.
```

File: content/blog/2021/07/webassembly-is-coming-to-cloud-native.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+---
+
+{{< figure class="center" src="/images/wasmcloud-logo.png" width="60%" alt="wasmCloud logo">}}
+
+[wasmCloud](https://wasmcloud.com/) is a project with a strong focus on the
+developer experience. wasmCloud provides a way to write portable code, code that can be
```
Comment (ereslibre) [inline content/blog/2021/07/webassembly-is-coming-to-cloud-native.md:84]:
```suggestion
developer experience. wasmCloud provides a way to write portable code. Code that can be
```

File: content/blog/2021/07/webassembly-is-coming-to-cloud-native.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+On the surface this project doesn't seem to be related with WebAssembly. However,
+without ORAS, neither Krustlet nor Kubewarden could store WebAssembly modules
+inside of OCI Container registries.
+
+---
+
+On behalf of the Kubewarden developers, let me congratulate with all the teams
+behind these projects. Thanks for having inspired us and for having paved the
+way of WebAssembly inside of CNCF!
```
Comment (ereslibre) [inline content/blog/2021/07/webassembly-is-coming-to-cloud-native.md:109]: This is so cool. :+1:

**Review by ereslibre — COMMENTED** [review]
Have tried it locally and all resources are looking fine. The content is awesome too. Some comments, thanks!

File: content/blog/2021/07/webassembly-is-coming-to-cloud-native.md (hunk 5 above)
Comment (flavio) [reply content/blog/2021/07/webassembly-is-coming-to-cloud-native.md:84]: CK is reviewing the doc too. I've seen she has changed this section in a different way. I'll propagate her change

**Pull request merged**